packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/save-event-to-db.job.ts

```diff
@@ -33,22 +34,35 @@ export class SaveEventToDbJob implements MessageQueueJob<SaveEventToDbJobData> {
     const eventType = `${data.operation}.${data.objectName}`;
 
     // TODO: add "workspaceMember" (who performed the action, need to send it in the event)
-    // TODO: need to support objects others than "person", "company", "opportunities"
+    // TODO: need to support objects others than "person", "company", "opportunity"
 
     if (
       data.objectName != 'person' &&
       data.objectName != 'company' &&
-      data.objectName != 'opportunities'
+      data.objectName != 'opportunity'
     ) {
       return;
     }
 
+    if (data.details.diff) {
+      data.details = {
+        diff: data.details.diff,
+      };
+    }
+
+    const workspaceMember = await workspaceDataSource?.query(`
```

Comment:
Using prepared param
```suggestion
    const workspaceMember = await workspaceDataSource?.query(`
          SELECT "id" FROM ${dataSourceMetadata.schema}."workspaceMember"
      WHERE ("userId" = $1) LIMIT 1;
      `, [data.userId])
```

Reply:
Also you can now move this logic to repositories inside `packages/twenty-server/src/modules/workspace-member/repositories/workspace-member.repository.ts`
To inject it 
```
    @InjectObjectMetadataRepository(WorkspaceMemberObjectMetadata)
    private readonly workspaceMemberService: WorkspaceMemberRepository,
```
then import this module
```
ObjectMetadataRepositoryModule.forFeature([
      WorkspaceMemberObjectMetadata,
    ]),
```